From d874d7d2ef9b9a57c7a1554ca8241b6402567932 Mon Sep 17 00:00:00 2001
From: Radivoje Jovanovic <radivoje.jovanovic@intel.com>
Date: Sat, 17 May 2014 09:04:19 -0700
Subject: [PATCH] remove i2c address from the driver name

Due to changes to i2c-core i2c address is no
longer needed. If preserved, the driver will
not load correctly

For: GMIN-856
Change-Id: Ic71f356188d0e42427b7bb6c165d816604520ef0
Signed-off-by: Radivoje Jovanovic <radivoje.jovanovic@intel.com>
---
 sound/soc/intel/byt-rt5640.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/byt-rt5640.c b/sound/soc/intel/byt-rt5640.c
index 8075c71..dbc63d0 100644
--- a/sound/soc/intel/byt-rt5640.c
+++ b/sound/soc/intel/byt-rt5640.c
@@ -119,7 +119,7 @@ static struct snd_soc_dai_link byt_rt5640_dais[] = {
 		.stream_name = "Audio",
 		.cpu_dai_name = "baytrail-pcm-audio",
 		.codec_dai_name = "rt5640-aif1",
-		.codec_name = "i2c-10EC5640:00:1c",
+		.codec_name = "i2c-10EC5640:00",
 		.platform_name = "baytrail-pcm-audio",
 		.dai_fmt = SND_SOC_DAIFMT_I2S | SND_SOC_DAIFMT_NB_NF |
 			   SND_SOC_DAIFMT_CBS_CFS,
-- 
1.9.3

